aw-server/Cargo.toml

```diff
@@ -28,7 +28,7 @@ fern = { version = "0.6", features = ["colored"] }
 toml = "0.5"
 gethostname = "0.2"
 uuid = { version = "0.8", features = ["serde", "v4"] }
-getopts = "0.2"
+clap = "3.0.0-beta.2"
```

Comment:
I chose getops over clap due to getopts being so much leaner, clap has a silly amount of dependencies for just being a options parser.
Personally I don't think all those extra dependencies are worth the very small improvements that clap gives for our use-case.

Reply:
Oh, didn't know that. 

I just find working with clap/clicky-like CLI builders a lot easier, the resulting CLI is better, and it makes the code simpler (but at the cost of dependencies).